decidim-proposals/app/controllers/decidim/proposals/proposal_votes_controller.rb

```diff
@@ -29,13 +30,18 @@ def destroy
       # The vote buttons should not be visible if the setting is not enabled.
       # This ensure the votes cannot be created using a POST request directly.
       def check_current_settings!
-        render nothing: true, status: 422 and return unless current_settings.votes_enabled?
+        head(422) and return if !current_settings.votes_enabled? || current_settings.votes_blocked?
```

Comment:
Are you sure this is the best way to deal with this? Shouldn't we declare a generic exception and then use `rescue_from` from the application controller?